src/tests/unit/services/AuthorizationRequestProcessor.test.ts

```diff
@@ -64,11 +64,10 @@ describe("AuthorizationRequestProcessor", () => {
 			redirect_uri: "http://localhost:8085/callback",
 			state: "Y@atr",
 		}));
-
 		// eslint-disable-next-line @typescript-eslint/unbound-method
 		expect(mockF2fService.setAuthorizationCode).toHaveBeenCalledTimes(1);
 		// eslint-disable-next-line @typescript-eslint/unbound-method
-		expect(mockF2fService.sendToTXMA).toHaveBeenCalledTimes(1);
+		expect(mockF2fService.sendToTXMA).toHaveBeenCalledTimes(2);
```

Comment:
Please can we have a `toHaveBeCalledIWith` to make sure we're sending the right event?